display/randr.go

```diff
@@ -6,12 +6,15 @@ import (
 	"reflect"
 	"strings"
 
-	"github.com/lpicanco/i3-autodisplay/i3"
+	"github.com/jezek/xgb"
 
-	"github.com/BurntSushi/xgb"
-	"github.com/BurntSushi/xgb/randr"
-	"github.com/BurntSushi/xgb/xproto"
+	"github.com/jezek/xgb/randr"
+	"github.com/jezek/xgb/xproto"
 	"github.com/lpicanco/i3-autodisplay/config"
+	"github.com/lpicanco/i3-autodisplay/i3"
+	// "github.com/BurntSushi/xgb/randr"
+	// "github.com/BurntSushi/xgb/xproto"
+	// "github.com/lpicanco/i3-autodisplay/config"
```

Comment:
chore: can you remove these commented lines?

Reply:
Hi,
thank you for your feedback.
I’ve removed the three lines as requested.